testapi.pm

```diff
@@ -2335,4 +2335,15 @@ A typical call would look like:
 
 sub show_curl_progress_meter { get_var('UPLOAD_METER') ? "-o /dev/$serialdev " : '' }
 
+=head2 backend_need_extra_wait_on_script_output
+
+Function to query the backend, if it has the known bug from poo#60566, when
+typing to fast into the here-script intput, the script content becomes wrong.
+=cut
+sub backend_need_extra_wait_on_script_output
```

Comment:
```suggestion
sub backend_needs_extra_wait_on_script_output
```

Reply:
I changed the name to `backend_get_sleep_on_here_script_intput()`